parachains/runtimes/asset/statemine/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright Parity Technologies (UK) Ltd.
```

Comment:
Looks like the bot is creating a *new* file  
`parachains/runtimes/asset/statemine/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs` vs   
`parachains/runtimes/assets/statemine/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs`.  
Note the `asset` vs `assets`.

Reply:
sorry, that's a typo in docs, the correct kind is `assets`, fixed in docs: https://github.com/paritytech/command-bot-scripts/pull/18/files